tests/e2e/docker/image_config.go

```diff
@@ -0,0 +1,64 @@
+package docker
+
+// ImageConfig contains all images and their respective tags
+// needed for running e2e tests.
+type ImageConfig struct {
+	InitRepository string
+	InitTag        string
+
+	OsmosisRepository string
+	OsmosisTag        string
+
+	RelayerRepository string
+	RelayerTag        string
+}
+
+const (
+	// Local osmosis repo/version.
+	// It is used when skipping upgrade by setting OSMOSIS_E2E_SKIP_UPGRADE to true).
+	// This image should be pre-built with `make docker-build-debug` either in CI or locally.
+	LocalOsmoRepository = "osmosis"
+	LocalOsmoTag        = "debug"
```

Comment:
Because we need to reuse it outside of the package for the upgrade:
https://github.com/osmosis-labs/osmosis/blob/3ddd10a7e3e36273628efe9c199021d42013613f/tests/e2e/e2e_setup_test.go#L553